docs/section11/Rule_11-1.md

```diff
@@ -0,0 +1,65 @@
+# Service_Water_Heating - Rule 11-1
+**Schema Version:** 0.0.36  
+
+**Mandatory Rule:** TRUE
+
+**Rule ID:** 11-1
+
+**Rule Description:** "Where a complete service water-heating system exists, the proposed design shall reflect the actual system type. Where a service water-heating system has been designed the service waterheating type shall be consistent with design documents."
+
+**Rule Assertion:** Options are PASS/FAIL
+
+**Appendix G Section Reference:** Table G3.1 #11, proposed column, a & b
+
+**Evaluation Context:** P-RMD each SWH Distribution
+**Data Lookup:**   
+**Function Call:** 
+- **get_component_by_id**
+- **compare_context_pair** - there is no RDS for this function, but it is a function developed for Rule 1-6 that compares two elements
+- **get_SWH_types_and_spaces**
+- **get_SWH_equipment_associated_with_each_swh_distribution_system**  
+
+**Applicability Checks:**
+- check that the U_RMD has SWH Uses with loads
+
+## Applicability Checks:
+- only projects with SWH in the user model are expected to have a SWH system in the proposed model.  If there is no SWH in the user model, we assume there is no SWH system designed, and instead rule 11-3 Applies and P_RMD matches B_RMD.
+- use the function get_SWH_equipment_associated_with_each_swh_distribution_system to get a dictionary of SWH equipment associated with each Distribution System for the Proposed model (pumps, tanks, SWH use, distribution system, etc): `p_swh_system_and_equip_dict = get_SWH_equipment_associated_with_each_swh_distribution_system(P_RMD)`
+- use the function get_SWH_equipment_associated_with_each_swh_distribution_system to get a dictionary of SWH equipment associated with each Distribution System for the User model (pumps, tanks, SWH use, distribution system, etc): `u_swh_system_and_equip_dict = get_SWH_equipment_associated_with_each_swh_distribution_system(U_RMD)`
+- for each distribution system: `for distribution_id in p_swh_system_and_equip_dict:`
+    - get the distribution system: `p_distribution = get_component_by_id(P_RMD, distribution_id)`
+    - for each swh use in the dictionary, check to see that the swh use exists in the user model: `for swh_use_id in p_swh_system_and_equip_dict[distribution_id]["SPACE_USES"]:`
+        - get the user swh_use: `u_swh_use = get_component_by_id(U_RMD, swh_use_id)`
+        - if the u_swh_use exists, continue to rule logic as we only need one SWH use to have this rule be applicable: `if(u_swh_use): CONTINUE TO RULE LOGIC`
```

Comment:
So this will keep repeating the logic below for all uses?

Reply:
yes, for all object types in included in [SHW comparison fields.yaml](https://github.com/pnnl/ruleset-checking-tool/pull/843/files#diff-e7ef3fbae75b2279d8581158c48c728c60850fa1e3b3101d3b4c613fad81ef4e)

Reply:
I am not following why this all should be repeated for each use